configure

```diff
@@ -3865,6 +3896,14 @@ if test -n "$_host"; then
 			_seq_midi=no
 			_timidity=no
 			;;
+		sailfish)
+                        append_var DEFINES "-DSAILFISH"
+                        append_var LDFLAGS "-Wl,-rpath,/usr/share/org.scummvm.scummvm/lib"
```

Comment:
One of the problems with ScummVM on iOS is that somebody registered our application domain, thus, we must use a different one. Is this hardcoding of `org.scummvm.scummvm` safe? Or, perhaps, there should be two settings, for local build and for the release build?